bin/npack

```diff
@@ -88,6 +95,13 @@ program.command('install <source>')
 	.action(function(src, options) {
 		Steppy(
 			function() {
+				if (!_(['install', 'ci']).contains(options.syncMode)) {
+					throw new Error(
+						'Expect sync mode "' + options.syncMode + '" to be ' +
```

Comment:
suggestion: to not manually enumerate all possible values in error text (throw unknown sync mode error or get values from variable)